channels/us.m3u

```diff
@@ -5493,7 +5493,7 @@ https://dai2.xumo.com/amagi_hls_data_xumo1212A-redboxsamuelgoldwyn/CDN/playlist.
 https://samuelgoldwyn-films-roku.amagi.tv/playlist.m3u8
 #EXTINF:-1 tvg-id="SamuelGoldwynClassics.us" tvg-name="Samuel Goldwyn Classics" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/oCjZS0v.jpg" group-title="Classic",Samuel Goldwyn Classics
 https://samuelgoldwyn-classic-roku.amagi.tv/playlist.m3u8
-#EXTINF:-1 tvg-id="SamuraiJack.us" tvg-name="Samurai Jack (Adult Swim 24/7 Stream)" tvg-country="US" tvg-language="English" tvg-logo="" group-title="",Samurai Jack (Adult Swim 24/7 Stream) [Geo-blocked]
+#EXTINF:-1 tvg-id="SamuraiJack.us" tvg-name="Samurai Jack (Adult Swim 24/7 Stream)" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/zIjbwSa.jpg" group-title="Series",Samurai Jack (Adult Swim 24/7 Stream) [Geo-blocked]
```

Comment:
I think all of those series from Adult Swim fall into Animation category

Reply:
Okay I change those.